src/Standards/Generic/Docs/Commenting/DocCommentStandard.xml

```diff
@@ -0,0 +1,394 @@
+<documentation title="Doc Comment">
+    <standard>
+    <![CDATA[
+    Enforces several rules related to the formatting of doc comments in PHP code.
+    ]]>
+    </standard>
+    <standard>
+    <![CDATA[
+    Doc comment must not be empty.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Doc comment with some content.">
+        <![CDATA[
+/**
+ * <em>Some content.</em>
+ */
+        ]]>
+        </code>
+        <code title="Invalid: Empty doc comment.">
+        <![CDATA[
+/**
+ * <em></em>
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    The opening doc comment tag must be the only content on the line.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: The opening doc comment tag is the only content on the line.">
+        <![CDATA[
+<em>/**</em>
+ * Short description.
+ */
+        ]]>
+        </code>
+        <code title="Invalid: The opening doc comment tag is not the only content on the line.">
+        <![CDATA[
+<em>/** Short description.</em>
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    The closing doc comment tag must be the only content on the line.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: The closing doc comment tag is the only content on the line.">
+        <![CDATA[
+/**
+ * Short description.
+ <em>*/</em>
+        ]]>
+        </code>
+        <code title="Invalid: The closing doc comment tag is not the only content on the line.">
+        <![CDATA[
+/**
+ <em>* Short description */</em>
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    There must be no additional blank lines before the closing doc comment tag.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: no additional blank lines before the closing doc comment tag.">
+        <![CDATA[
+/**
+ * Short description.
+<em></em> */
+        ]]>
+        </code>
+        <code title="Invalid: additional blank lines before the closing doc comment tag.">
+        <![CDATA[
+/**
+ * Short description.
+<em> *</em>
+ */
+
+ /**
+ * Short description.
+<em></em>
+ */
```

Comment:
This seems like a stray example. I realize that you are highlighting with and without the `*`, but IMO, that isn't really a significant enough difference to have a second code sample.